UPGRADE-3.x.md

```diff
@@ -65,6 +65,118 @@ return [
 +    Sulu\Page\Infrastructure\Symfony\HttpKernel\SuluPageBundle::class => ['all' => true],
 ```
 
+Then you need to update the route configuration in your `config/routes/sulu_admin.yaml`:
+
+```diff
+ sulu_snippet_api:
+-    resource: "@SuluSnippetBundle/Resources/config/routing_api.yml"
+-    type: rest
++    resource: "@SuluSnippetBundle/config/routing_admin_api.yaml"
+     prefix: /admin/api
+
+# TODO sulu_page_api or sulu_next_page_api ?
++sulu_next_page_api:
```

Comment:
Not sure how we handle this currently because according to this upgrade.md we do already remove the legacy SuluPageBundle from the bundles.php 😅

Reply:
As discussed, the upgrade md should work for the current alpha release, as we will update it later on as soon as we have a stable 3.0